python/doc/examples/functional_modeling/link_computer_code_coupling_tools.ipynb

```diff
@@ -0,0 +1,459 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Link to a computer code with coupling tools"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this example we show how to use the `coupling_tools` module, which allows to create a function from a computer code based on text file exchanges. We show the main features of the module on a simple example and focus on the `replace` and `get` functions."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "\n",
+    "The `coupling_tools` module is useful when the external computer code reads (on input) and write (on output) text files. \n",
+    "\n",
+    "<img src=\"link_computer_code.svg\" width=\"600px\">\n",
+    "\n",
+    "The main features of the `coupling_tools` module are:\n",
+    "\n",
+    "* `replace`: writes a file based on a file template, by replacing tokens with values,\n",
+    "* `execute`: executes an external computer code,\n",
+    "* `get` (and `get_line_col`): reads values from a file.\n",
+    "\n",
+    "Moreover, the `coupling_tools` module can be useful outside the library, for example to evaluate a design of experiments on a cluster. \n",
+    "\n",
+    "There are several advantages over basic Python scripting while using the module:\n",
+    "\n",
+    "* it is much more simpler than using regular expressions, \n",
+    "* it allows to skip lines, colonnes or blocks of text.\n",
+    "\n",
+    "It is easy to use if the input or output files are based on structured text files. It is possible to parallelize it, with a little more Python scripting."
```

Comment:
```suggestion
    "It is easy to use if the input and output files are based on structured text files. With a little more Python scripting, it is even possible to parallelize it."
```

Reply:
No, it is a *or*, because we can use only a part of the API, not necessarily all of it. The second sentence is OK: I integrated it.